### DISCUSSION TITLE
ALT+{1,2,3,4,5,6,7,8} not sent to applications acts like SUPER+{1,2,3,3,4,5,6,7,8,9} instead

### DISCUSSION THREAD

**Discussion by lakinwecker** [wezterm/wezterm#1649] [Q&A] [answered]
If I turn on `disable_default_key_bindings` then it's sent to the applications.  Running wezterm with `wezterm --config debug_key_events=true` shows that it's receiving the keycodes correctly: https://gist.github.com/lakinwecker/63e20ed486d58ed25ab2eb34fec9da33

But neither the terminal, nor `od -c` are reporting them.  I've looked at the default key assignments here: https://github.com/wez/wezterm/blob/main/config/src/keyassignment.rs#L384-L723 but I'm not seeing anything that looks obvious. 

Edit:
Whoops, forgot to include the version number in this last one:
```➜ wezterm --version
wezterm 20220101-133340-7edc5b5a
```

**Comment by wez** [comment, marked as answer]
Released versions of wezterm had a default assignment for ALT+number, but that assignment has been removed in `main`.
Please try the nightly build!

**Reply by lakinwecker** [reply]
Ok, will do, thanks!
